JenkinsfileRT

```diff
@@ -73,7 +73,6 @@ bc0 = new BuildConfig()
 bc0.nodetype = 'jwst'
 bc0.name = 'stable-deps'
 bc0.env_vars = env_vars
-bc0.conda_ver = '4.8.4'
```

Comment:
Removing this means the default conda version will be used, which might be out-of-date.  We should probably just bump this to the latest for now until we fix the Jenkins CI helpers to get the latest, i.e.

https://github.com/spacetelescope/jenkins_shared_ci_utils/blob/37606ed9c9a9133e7983183e52976683d8d25ece/vars/utils.groovy#L106-L108

Reply:
Oops, yes, I have a PR open over there (https://github.com/spacetelescope/jenkins_shared_ci_utils/pull/80) but I agree we should wait until that's actually merged.